packages/app/src/components/Root/Root.tsx

```diff
@@ -101,10 +102,19 @@ const getMenuItem = (menuItem: ResolvedMenuItem, isNestedMenuItem = false) => {
   );
 };
 
+const GlobalHeader = ({ mountPoints }: { mountPoints: MountPoints }) =>
+  mountPoints['application/header']?.map(({ Component, config }) => (
+    <Component key="global-header" {...config?.props} />
+  ));
+
 export const Root = ({ children }: PropsWithChildren<{}>) => {
-  const { dynamicRoutes, menuItems } = useContext(DynamicRootContext);
+  const { dynamicRoutes, menuItems, mountPoints } =
+    useContext(DynamicRootContext);
   const [openItems, setOpenItems] = useState<{ [key: string]: boolean }>({});
 
+  const { position: globalHeaderPosition } =
+    mountPoints['application/header']?.[0]?.config?.layout ?? {};
```

Comment:
Let us allow both headers also in parallel instead of checking the first config layout value.